### PR TITLE
Liver Update: Beer Goggles edition

### PR DESCRIPTION
**Pull request by pyrosshade**
### Purpose
Implements reactions in the liver that convert alcohols into ethanal. This makes alcohol entering the body have more of an effect, adding time to the drunk camera for now.

### Notes:
LiverBreakdown Reactions have null effect slots for various alcohols at the moment in preparation for a future PR 
ChemHallucination is incomplete and a placeholder
Conversion values somewhat mimic tg boozepwr values though may be subject to change
Values in the liver regarding how the liver performs have been ballpark tweaked and will need future adjustment to dial in 
TODO: change drunkenness to transition through stages.
TODO: implement adverse effects for being drunk for too long (reaching later stages)

### Changelog:
CL: [New] Liver Alcohol processing expanded. If Alcohol reaches the liver faster than the liver can filter it out, time gets added to the drunk effects.
CL: [Balance] Alcohols now have varying strengths, resulting in similiar volumes having differing amounts of time being drunk.

### CODE REVIEW

**Comment by pyrosshade** [comment]
@MoonTheBird I must make note that this pr does not implement anything to do with the Alien Cuisine alcohols. When/if this does merge, if you need help with implementing your additions to function similiarly then gimme a poke.

**Comment by VernaSaurus** [comment]
I can implement them myself, don't worry.

File: UnityProject/Assets/Scripts/Items/Implants/Organs/Liver.cs [hunk 1]
```diff
@@ -37,44 +41,64 @@ public class Liver : BodyPartFunctionality
...
-			float tickPullProcessingAmnt =  RelatedPart.TotalModified *  processAmount;
-			float tickClearAmount = tickPullProcessingAmnt * flushMultiplier;
-
-			ReagentContainerBody blood = RelatedPart.BloodContainer;
+			BloodToLiver(blood);
+			Processing();
+			ReturnReagentsToBlood();
 
-			List<Tuple<Reagent,float>> tempArray = new List<Tuple<Reagent, float>>();
+			Logger.Log(debug.ToString(), Category.Health);
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Items/Implants/Organs/Liver.cs]: Log and debug stringbuilder still needed? Comment them out if you think they'll be useful in the future

File: UnityProject/Assets/Scripts/Items/Food/DrinkableContainer.cs [hunk 2]
```diff
@@ -79,11 +81,27 @@ public override void TryConsume(GameObject feederGO, GameObject eaterGO)
...
-
-		DoDrinkEffects(eater, drinkAmount);
+		int stomachIndex = 0;
+		List<Stomach> stomachs = eater.playerHealth.GetStomachs();
+		while (drinkAmount > 0)
+		{
+			ReagentContainer stomachContainer = stomachs[stomachIndex].StomachContents;
+			if (stomachContainer.IsFull)
+			{
+				if (stomachIndex < stomachs.Count-1)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Food/DrinkableContainer.cs]: what does this do?

File: UnityProject/Assets/Scripts/Items/Implants/Organs/Liver.cs (hunk 1 above)
Comment (corp-0) [reply UnityProject/Assets/Scripts/Items/Implants/Organs/Liver.cs]: it is currently logging an empty string, it is safe to just remove

Comment (corp-0) [reply UnityProject/Assets/Scripts/Items/Implants/Organs/Liver.cs]: ah I opened my mouth before continuing reading

File: UnityProject/Assets/Scripts/Items/Food/DrinkableContainer.cs (hunk 2 above)
Comment (pyrosshade) [reply UnityProject/Assets/Scripts/Items/Food/DrinkableContainer.cs]: Transfers the reagents in the DrinkableContainer to each stomach as the preceding stomach fills up until the drink is empty. Currently rewriting this on local.

**Comment by pyrosshade** [comment]
Also, now with feedback for when you can't stuff more alcomahols down your stomach-gullet.

**Review by corp-0 — APPROVED** [review]

**Comment by pyrosshade** [comment]
Big oopsie in removing old branches

**Review by NoooneyDude — APPROVED** [review]

**Pull request merged**